RetsTalk/RetsTalk/RetrospectCalendar/Controller/RetrospectCalendarViewController.swift

```diff
@@ -114,7 +147,15 @@ extension RetrospectCalendarViewController: @preconcurrency UICalendarViewDelega
 
 extension RetrospectCalendarViewController: @preconcurrency UICalendarSelectionSingleDateDelegate {
     func dateSelection(_ selection: UICalendarSelectionSingleDate, didSelectDate dateComponents: DateComponents?) {
-        print("선택된 날자: \(String(describing: dateComponents))")
+        guard let dateComponents = dateComponents else {
+            print("선택된 날짜가 없습니다.")
+            return
+        }
```

Comment:
```suggestion
        guard let dateComponents = dateComponents
        else {
            print("선택된 날짜가 없습니다.")
            return
        }
```

print문은 alert로 알릴 건가요 ?

Reply:
클릭시 alert 계속 뜨는건 좋지 않아서 그냥 return 해주는 형식으로 할 것 같습니다!